File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/teleops/BlueTeleop.java
package org.firstinspires.ftc.teamcode.teleops;

import com.qualcomm.robotcore.eventloop.opmode.TeleOp;

@TeleOp(name = "BlueTeleop.j")
public class BlueTeleop extends JavaCompetitionTeleop{

    @Override
    protected AllianceColor getAllianceColor() {
       return AllianceColor.BLUE;
    }
}

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/teleops/BlueTeleop.kt
package org.firstinspires.ftc.teamcode.teleops

import com.qualcomm.robotcore.eventloop.opmode.TeleOp

@TeleOp(name = "BlueTeleop")
class BlueTeleop : CompetitionTeleop() {
    override val allianceColor: AllianceColor = AllianceColor.BLUE
}